os_site/src/_includes/post.njk

```diff
@@ -0,0 +1,39 @@
+---
+layout: base.njk
+---
+<main>
+  <article>
+    <div class="post__title t-align-center t-serif">
+        <h1>{{ title }}</h1>
+    </div>
+    <p class="t-byline t-align-center has-text-gray">
+      <span class="t-byline__item l-display-ib">by {{ author }} </span>
+      <time class="t-byline__item l-display-ib"> {{ date | readableDate }}</time>
+    </p>
+    <figure class="post__image">
+        <img src="{{ image_url | url}}" alt="{{ image_alt }}" >
+    </figure>
+    <div class="post__content has-xl-padding t-serif">
+        {{ content | safe }}
+    </div>
+  </article>
+</main>
+
+<nav aria-label="post" class="post-nav-container t-links t-uppercase t-lsp-b t-size-xs t-align-center t-weight-bold">
```

Comment:
It might be that these styles don't work here for whatever reason, but there is a [queso component for pagination](https://texastribune.github.io/queso-ui/sections/components/c-pagination/) if that helps eliminate any need for custom styles.

It might make sense to use something more descriptive for the aria-label like `aria-label="pagination"`

Reply:
Thank you! I did find that queso component but found it difficult to style at the time. Making a note to come back to this later today and try again